src/utils/index.es

```diff
@@ -0,0 +1,13 @@
+import React, { forwardRef } from 'react';
+
+export const getDisplayName = (wrapper, Component) => {
```

Comment:
`getProxyDisplayName | getWrapperDisplayName`?

Reply:
Same thing tho isnt it?

Reply:
maybe getPrefixedDisplayName?